NEARFlags.json

```diff
@@ -0,0 +1,31 @@
+{
+  "EXAMPLE_FLAG": {
+    "createdBy": "Daryl Collins",
+    "createdAt": "2021-05-24T09:33:27.489Z",
+    "development": {
```

Comment:
do I understand correctly that this environment gonna get used for PR deploys?

Reply:
I was thinking this would be code that should only be on for the developer's local environments (not yet running in any other environment).

The problem I've seen with our PR builds is that they can create a chicken-and-egg problem for required backend code changes that are required for the new functionality in a PR.  With feature flags like this, we could probably eliminate PR builds and make the equivalent to one of our current PR builds that is ready for testing be 'I flagged it on in `testnet-staging` and merged it', so that it's clear from the codebase what code is going to be running against which environment.

We could also retain PR builds/deploy, but make them use `testnet-staging`, which is closest equivalent to what we do now.  However, it's really up to us; we could even do N number of builds from any PR that are configured for whichever environments we'd like to be able to test from the PR (e.g. we could do a build for mainnet staging and testnet staging on every PR if we want), but see my above concerns about crossing environment boundaries and not having a clear idea of what code is running against the backend.